profile/profile.go

```diff
@@ -103,18 +110,55 @@ func StartContinuousProfile(profileDir string) {
 		// Collect statistics in an infinite loop.
 		sleepTime := time.Second * 20
 		for {
+			StopCPUProfile()
+
+			// Print the mem profile.
+			SaveMemProfile(profileDir, "continuousProfilingMem")
+
+			// Log the mem stats.
+			var m runtime.MemStats
+			runtime.ReadMemStats(&m)
+			b := new(bytes.Buffer)
+			w := tabwriter.NewWriter(b, 0, 0, 2, ' ', 0)
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "Goroutines:\t", humanize.Comma(int64(runtime.NumGoroutine())))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "Sys (RES):\t", humanize.Bytes(m.Sys))
+			fmt.Fprintln(w, "HeapSys:\t", humanize.Bytes(m.HeapSys))
+			fmt.Fprintln(w, "StackSys:\t", humanize.Bytes(m.StackSys))
+			fmt.Fprintln(w, "MSpanSys:\t", humanize.Bytes(m.MSpanSys))
+			fmt.Fprintln(w, "MCacheSys:\t", humanize.Bytes(m.MCacheSys))
+			fmt.Fprintln(w, "BuckHashSys:\t", humanize.Bytes(m.BuckHashSys))
+			fmt.Fprintln(w, "GCSys:\t", humanize.Bytes(m.GCSys))
+			fmt.Fprintln(w, "OtherSys:\t", humanize.Bytes(m.OtherSys))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "TotalAlloc:\t", humanize.Bytes(m.TotalAlloc))
+			fmt.Fprintln(w, "Alloc:\t", humanize.Bytes(m.Alloc))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "HeapObjects:\t", humanize.Comma(int64(m.HeapObjects)))
+			fmt.Fprintln(w, "HeapSys:\t", humanize.Bytes(m.HeapSys))
+			fmt.Fprintln(w, "HeapAlloc:\t", humanize.Bytes(m.HeapAlloc))
+			fmt.Fprintln(w, "HeapIdle:\t", humanize.Bytes(m.HeapIdle))
+			fmt.Fprintln(w, "HeapInuse:\t", humanize.Bytes(m.HeapInuse))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "StackSys:\t", humanize.Bytes(m.StackSys))
+			fmt.Fprintln(w, "StackInuse:\t", humanize.Bytes(m.StackInuse))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "MSpanSys:\t", humanize.Bytes(m.MSpanSys))
+			fmt.Fprintln(w, "MSpanInuse:\t", humanize.Bytes(m.MSpanInuse))
+			fmt.Fprintln(w)
+			fmt.Fprintln(w, "MCacheSys:\t", humanize.Bytes(m.MCacheSys))
+			fmt.Fprintln(w, "MCacheInuse:\t", humanize.Bytes(m.MCacheInuse))
```

Comment:
Is this the best format for this data? I figured you would want it in some sort of tabular form so that you can easily see how a given value changes over time. E.g.
```
Goroutines    Alloc
100           10
150           20
125           30
```
etc.